spec/support/driver_helper.rb

```diff
@@ -13,7 +13,7 @@ def uri
 
   class << self
     def uri
-      ENV.fetch('NEO4J_BOLT_URL', 'bolt://localhost:7687')
+      ENV.fetch('NEO4J_BOLT_URL', 'bolt://localhost:7472')
```

Comment:
I intentionally changed the port to prevent overwriting the org-api dev database